ticdc/troubleshoot-ticdc.md

```diff
@@ -347,6 +347,35 @@ Since v4.0.8, if the `canal` or `maxwell` protocol is used for output in a chang
     cdc cli changefeed resume -c test-cf --pd=http://10.0.10.25:2379
     ```
 
+## The `[tikv:9006]GC life time is shorter than transaction duration, transaction starts at xx, GC safe point is yy` error is reported when I use TiCDC to create a changefeed
+
+Solution: You need to execute the `pd-ctl service-gc-safepoint --pd <pd-addrs>` command to query the current GC safepoint and service GC safepoint. If the GC safepoint is smaller than the `start-ts` of the TiCDC replication task (changefeed), you can directly add the `--disable-gc-check` option to the `cdc cli create changefeed` command to create a changefeed.
+
+If the result of `pd-ctl service-gc-safepoint --pd <pd-addrs>` does not have `gc_worker service_id`:
+
+- If your PD version is v4.0.8 or earlier, refer to [PD issue #3128](https://github.com/tikv/pd/issues/3128) for details.
+- If your PD is upgraded from v4.0.8 or an earlier version to a later version, refer to [PD issue #3366](https://github.com/tikv/pd/issues/3366) for details.
+- For other situations, report the execution result of the above command to [AskTUG](https://asktug.com/tags/ticdc).
```

Comment:
@lichunzhu Any alternative to AskTUG is English version? AskTuG is only for Chinese.

Reply:
```suggestion
- For other situations, report the execution result of the above command to [Slack](https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=pingcap-community)
```